plugins/Super_Saw/Makefile

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/make -f
+# Makefile for DISTRHO Plugins #
+# ---------------------------- #
+# Created by falkTX
+#
+
+# --------------------------------------------------------------
+# Project name, used for binaries
+
+NAME = Super_Saw
+
+# --------------------------------------------------------------
+# Files to build
+
+OBJS_DSP = \
+	SuperSaw.cpp.o
+
+#OBJS_UI  = \
```

Comment:
are you planning to add an UI? or not needed?
if not, please remove the old stuff here.